src/py/mat3ra/made/cell/cell.py

```diff
@@ -1,41 +1,45 @@
 from typing import List
 
 import numpy as np
-from mat3ra.esse.models.core.primitive.array_of_3_numbers import ArrayOf3NumberElementsSchema
 from mat3ra.utils.mixins import RoundNumericValuesMixin
 from pydantic import BaseModel
 
 
 class Cell(RoundNumericValuesMixin, BaseModel):
-    # TODO: figure out how to use
-    vector1: ArrayOf3NumberElementsSchema = [1, 0, 0]
-    vector2: ArrayOf3NumberElementsSchema = [0, 1, 0]
-    vector3: ArrayOf3NumberElementsSchema = [0, 0, 1]
+    # TODO: figure out how to use ArrayOf3NumberElementsSchema
+    vector1: List[float] = [1, 0, 0]
+    vector2: List[float] = [0, 1, 0]
+    vector3: List[float] = [0, 0, 1]
     __round_precision__ = 1e-6
 
     @classmethod
     def from_nested_array(cls, nested_array):
-        if not nested_array:
-            nested_array = [cls.vector1, cls.vector2, cls.vector3]
+        if nested_array is None:
+            nested_array = [[1, 0, 0], [0, 1, 0], [0, 0, 1]]
         return cls(vector1=nested_array[0], vector2=nested_array[1], vector3=nested_array[2])
 
-    def __init__(self, vector1=[1, 0, 0], vector2=[0, 1, 0], vector3=[0, 0, 1]):
+    def __init__(self, vector1=None, vector2=None, vector3=None):
+        if vector1 is None:
+            vector1 = [1, 0, 0]
+        if vector2 is None:
+            vector2 = [0, 1, 0]
+        if vector3 is None:
+            vector3 = [0, 0, 1]
         super().__init__(**{"vector1": vector1, "vector2": vector2, "vector3": vector3})
 
     @property
-    def vectors_as_array(self, skip_rounding=False) -> List[ArrayOf3NumberElementsSchema]:
+    def vectors_as_array(self, skip_rounding=False) -> List[List[float]]:
```

Comment:
The rounding logic should come from a mixin:

RoundMixin - has the round function
EntityRoundedMixin - has both `round`, `to_json` (with rounding for all array-like values) and `to_json_without_rounding`
Where you need to skip rounding, you'll use `to_json_without_rounding`

2. `vectors_as_nested_array` to be consistent with `from_nested_array`